### DISCUSSION TITLE
Welcome to LangChain-SearXNG Discussions!

### DISCUSSION THREAD

**Discussion by ptonlix** [ptonlix/LangChain-SearXNG#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by blastptah** [comment]
thank you!!! it is very well created!

**Reply by ptonlix** [reply]
You’re welcome! I’m glad you like it!
